src/Microsoft.ML.StandardTrainers/FactorizationMachine/FactorizationMachineInterface.netcoreapp.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics.X86;
+using System.Security;
+using Microsoft.ML.Internal.CpuMath;
+using Microsoft.ML.Runtime;
+
+namespace Microsoft.ML.Trainers
+{
+    internal static unsafe class FieldAwareFactorizationMachineInterface
+    {
+        internal const string NativePath = "FactorizationMachineNative";
+        public const int CbAlign = 16;
```

Comment:
We shouldn't be duplicating all this code. Instead, the pattern is to have 3 files:

1. a `netcoreapp.cs` file with the code that only applies to .NET Core.
2. a `netstandard.cs` file with the code that only applies to .NET Standard.
3. A `FactorizationMachine.cs` file that contains the common code between them.

Then the class is marked `partial`, so multiple files can add members to the class.